aayushi/src/components/Button.jsx

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable react/prop-types */
+
+export const Button = ({ loading, type, onClick }) => {
+  return (
+    <button className="  text-sm font-medium h-11 rounded-md w-full focus-visible:outline-none focus-visible:ring-2  max-w-sm mt-10 text-white bg-blue-800"  onClick={onClick}>
```

Comment:
Avoid using redundant space in classnames. Also, check if this button can be made a reusable component as it might get used at multiple places.